Cargo.lock

```diff
@@ -1358,7 +1358,7 @@ dependencies = [
  "rand_chacha 0.3.1",
```

Comment:
We should probably go with the approach from here: https://github.com/paritytech/polkadot-sdk/pull/4343 and pin the litep2p to a tag or sha, to avoid this type of breakage in the future.

Reply:
Yep, forgot about it when I named the branch `pin-litep2p-commit` 🙈 Thanks!